test/autoproj/daemon/test_overrides_retriever.rb

```diff
@@ -116,143 +99,117 @@ def create_pull_request(owner, name, number, body, state: "open")
                     )
                 end
             end
-            describe "#retrieve_dependencies" do
-                it "recursively fetches pull request dependencies" do
-                    body_driver_gps_ublox = <<~EOFBODY
-                        Depends on:
-                        - [ ] tidewise/drivers-orogen-gps_ublox#22
-                    EOFBODY
-                    pr_drivers_gps_ublox = add_pull_request(
-                        "tidewise", "drivers-gps_ublox",
-                        11, body_driver_gps_ublox
-                    )
-
+            describe "#dependencies" do
+                it "resolve the pull request dependencies" do
                     body_driver_orogen_gps_ublox = <<~EOFBODY
                         Depends on:
                         - [ ] rock-core/drivers-orogen-iodrivers_base#33
                         - [ ] tidewise/tidewise.common-package_set#44
                     EOFBODY
                     pr_driver_orogen_gps_ublox = add_pull_request(
-                        "tidewise", "drivers-orogen-gps_ublox",
-                        22, body_driver_orogen_gps_ublox
+                        "tidewise/drivers-orogen-gps_ublox", 22,
+                        body_driver_orogen_gps_ublox
                     )
-
                     pr_driver_orogen_iodrivers_base = add_pull_request(
-                        "rock-core", "drivers-orogen-iodrivers_base",
-                        33, nil
+                        "rock-core/drivers-orogen-iodrivers_base", 33, "iodrivers_base"
                     )
                     pr_package_set = add_pull_request(
-                        "tidewise", "tidewise.common-package_set",
-                        44, nil
+                        "tidewise/tidewise.common-package_set", 44, "package_set"
                     )
 
-                    depends = retriever.retrieve_dependencies(pr_drivers_gps_ublox)
-                    assert_equal [pr_driver_orogen_gps_ublox,
-                                  pr_driver_orogen_iodrivers_base,
+                    depends = retriever.dependencies(pr_driver_orogen_gps_ublox)
+                    assert_equal [pr_driver_orogen_iodrivers_base,
                                   pr_package_set], depends
                 end
-                it "breaks cyclic dependencies" do
```

Comment:
Could you keep this test case, please?

Reply:
Same answer. There is no recursive fetching since the dependency refactoring fetches flat and then PullRequest#recursive_dependencies resolve the recursion.

I'll move these test cases to PullRequest